### PR TITLE
Update db-auth.md

### PR DESCRIPTION
**Pull request by eugeneivanov101**
Signed-off-by: Eugene Ivanov <eivanov427@gmail.com> Replaced `disabled` with `deactivated` on line 22 (issue #303)

### CODE REVIEW

**Comment by vladdoster** [comment]
IMO, this makes the terminology incoherent since `enable(d)` is still used.

To remedy this, all uses of `enable(d)` should be changed to `activate(d)` since the negation of `deactivate(d)` is now used.

**Comment by vladdoster** [comment]
An example in the same line would be the word `register`. It would be confusing if we said `signed up` and `unregister`.

File: docs/administration/configure-authentication/db-auth.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ If you create users in the database, Harbor is locked in database mode. You cann
 
    ![Enable self-registration](../../../img/new-self-reg.png)
     
-   If you enable the self registration option, users can register themselves in Harbor. Self-registration is disabled by default. If you enable self-registration, unregistered users can sign up for a Harbor account by clicking **Sign up for an account** in the Harbor log in page.
+   If you enable the self registration option, users can register themselves in Harbor. Self-registration is deactivated by default. If you enable self-registration, unregistered users can sign up for a Harbor account by clicking **Sign up for an account** in the Harbor log in page.
```
Comment (vladdoster) [inline docs/administration/configure-authentication/db-auth.md:22]: These sentences basically repeat themselves, but a small nit (other than rewriting it altogether):

```suggestion
   If you enable the self registration option, users can register themselves in Harbor. Self-registration is deactivated by default. If you enable self-registration, unregistered users can create a Harbor account by clicking **Sign up for an account** in the Harbor log in page.
```

Comment (vladdoster) [reply docs/administration/configure-authentication/db-auth.md:22]:
```suggestion
   If you enable the self-registration option, users can register themselves in Harbor. Self-registration is deactivated by default. If you enable self-registration, unregistered users can sign up for a Harbor account by clicking **Sign up for an account** in the Harbor log in page.
```

**Comment by eugeneivanov101** [comment]
@vladdoster I'm making these changes as outlined in issue #303. I'm ready to take on all others this group finds appropriate.

**Pull request closed without merging**